simtools/layout/layout_array.py

```diff
@@ -940,3 +940,31 @@ def include_radius_into_telescope_table(telescope_table):
             for tel_name_now in telescope_table["telescope_name"]
         ]
         return telescope_table
+
+    def select_assets(self, asset_list=None):
+        """
+        Select a subsets of telescopes / assets from the layout.
+
+        Parameters
+        ----------
+        asset_list: list
+            List of assets to be selected.
+
+        Raises
+        ------
+        ValueError
+            If the asset list is empty.
+
+        """
+
+        try:
+            self._logger.info(
```

Comment:
This is a bit problematic to print to the log file before running the actual selection since if the next line leads to an exception the log file would be wrong. I suggest to combine this info line with the next one and put it after filling the telescope list.

Reply:
Agree.

Reply:
And I am not going to complain about using a different type of formatting than our usual f-string here (I assume copilot did this).

Reply:
I am tired that my editor shows errors using f-strings and logger. So don't complain.